keps/sig-scheduling/3521-pod-scheduling-readiness/README.md

```diff
@@ -805,12 +816,16 @@ Longer term, we may want to require automated upgrade/rollback tests, but we
 are missing a bunch of machinery and tooling and can't do that now.
 -->
 
+It will be tested manually prior to beta launch.
```

Comment:
This is fine, but please update the KEP with the scenario and results of that test (and cc me on it)  *before* switching the feature gate value to true in k/k.

Reply:
Leaving a <<UNRESOLVED>> section here to not block KEP's freeze date.